src/contrasts.jl

```diff
@@ -448,7 +448,22 @@ julia> StatsModels.hypothesis_matrix(seqdiff)
 """
 SeqDiffCoding
 
-function contrasts_matrix(C::SeqDiffCoding, baseind, n)
+mutable struct SeqDiffCoding <: AbstractContrasts
+    levels::Union{AbstractVector,Nothing}
+end
+function SeqDiffCoding(; base=nothing, levels::Union{AbstractVector,Nothing}=nothing)
+    if base !== nothing
+        Base.depwarn("`base=` kwarg for `SeqDiffCoding` is deprecated. " *
+                     "Specify full order of levels using `levels=` instead",
+                     :SeqDiffCoding)
```

Comment:
I would be tempted to be verbose here and write "and has no effect" to really emphasize that it's pointless